### PR TITLE
Update steps to fetch strings in release doc

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What

Update release doc to make it simpler it to fetch strings from TD Build and ensure the strings updates are not done on the release branch which never make it back to main.

# Why

https://skype.visualstudio.com/SPOOL/_workitems/edit/3235200

# How Tested

I will work with the next person performing the next release and monitor how effective or ineffective these updates will be.

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: .github/workflows/run-td-build.yml [hunk 1]
```diff
@@ -6,10 +6,6 @@ on:
   push:
     branches:
       - main
```
Comment (dmceachernmsft) [inline .github/workflows/run-td-build.yml:8]: there a reason for this yml edit? should we not be fetching strings from these other branches?

File: docs/releases/creating-a-release.md [hunk 2]
```diff
@@ -13,6 +13,11 @@ _This document applies to beta and stable releases. Alpha releases are created n
 
 _This document applies to normal releases, off of `main`. For hotfixing a prior release on NPM, see [documentation on creating hotfixes](./hotfix-release.md)._
 
+## Before creating a release branch
+
+- Update the strings on `main` these [steps](../references/string-translations.md).
+  - **Note**: String translation can take up to 5 working days to complete. It is a good idea to avoid string changes once before the release process
```
Comment (dmceachernmsft) [inline docs/releases/creating-a-release.md:19]: I thought the turn around was 10?

Comment (mgamis-msft) [reply docs/releases/creating-a-release.md:19]: No it's 5 :)

File: .github/workflows/run-td-build.yml (hunk 1 above)
Comment (mgamis-msft) [reply .github/workflows/run-td-build.yml:8]: I think all string changes should be done on main and be cherry-picked into the release branch. It's simpler.
String changes on release branches and the corresponding string translations don't ever make it back into main.

TD looks at our English strings on every push and has translations for each of them in their cache. If we have differing English strings in main vs a release branch, the translations in the cache will be on the latest push. I would like to have the translations always be for the English strings on main.

**Review by JamesBurnside — APPROVED** [review]

**Review by dmceachernmsft — APPROVED** [review]

**Pull request merged**